src/nrfx/hal/nrf_aar.h

```diff
@@ -0,0 +1,206 @@
+/**
+ * Part of the real nrf_aar.h from Nordic's NRFx HAL
+ *
+ * Note that there is a few changes compared to the original
+ * Where relevant, these changes are:
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
```

Comment:
A comment after an extremely quick look:
Please leave the original license and copyright text from Nordic below this one, as lost of the original file is copy pasted